seqr/management/commands/copy_project_tags.py

```diff
@@ -26,5 +28,5 @@ def handle(self, *args, **options):
             tag.id = None
             tag.project = target_project
             tag.save()
-            self.stdout.write('Saved tag %s (new id = %d)' % (tag.name, tag.id))
+            self.stdout._out.write('Saved tag %s (new id = %d)\n' % (tag.name, tag.id))
```

Comment:
same as my comment on the other PR, I think we should just switch this to logging

Reply:
Replaced stdio with logging.